projects/angular/src/utils/list-key-navigation/providers/key-navigation-list.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { InjectionToken, ValueProvider } from '@angular/core';
+
+export const ClrListKeyNavigationToken = new InjectionToken('ClrListKeyNavigation');
+
+export const ClrKeyNaviationListProvider = {
+  provide: ClrListKeyNavigationToken,
+  useValue: {},
+} as ValueProvider;
```

Comment:
What is this for? It doesn't seem to be injected anywhere.

Reply:
Good catch. Thanks.